webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
+  CleanWebpackPlugin = require('clean-webpack-plugin'),
+  HtmlWebpackPlugin = require('html-webpack-plugin'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const extractPlugin = new ExtractTextPlugin({filename: './assets/css/app.css'});
+
+const config = {
+
+  context: path.resolve(__dirname, 'src'),
+
+  entry: {
+    app: './index.js'
+  },
+
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: './assets/js/[name].bundle.js'
+  },
+
+  module: {
+    rules: [
+
+      {
+        test: /\.js$/,
+        include: /src/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ['env']
+          }
+        }
+      },
+      {
+        test: /\.html$/,
+        use: ['html-loader']
+      },
+      {
+        test: /\.s?css$/,
+        use: extractPlugin.extract({
+          use: [
+            {
+              loader: 'css-loader',
+              options: {
+                sourceMap: true
+              }
+            }, {
+              loader: 'sass-loader',
+              options: {
+                sourceMap: true
+              }
+            }
+          ],
+          fallback: 'style-loader'
+        })
+      },
+      {
+        test: /\.(jpg|png|gif|svg)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[name].[ext]',
+              outputPath: './assets/media/'
+            }
+          }
+        ]
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        use: ['file-loader']
+      }
+
+    ]
+  },
+
+  plugins: [
+    new CleanWebpackPlugin(['dist']),
+    new HtmlWebpackPlugin({template: 'index.html'}),
```

Comment:
```suggestion
    new HtmlWebpackPlugin({ template: 'index.html '}),
```

space inside curly braces